src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -133,7 +133,14 @@ export const AdminVenueView: React.FC = () => {
             venue={venue}
           />
         )}
-        {selectedTab === AdminVenueTab.run && <RunTabView venue={venue} />}
+        {selectedTab === AdminVenueTab.run && (
+          <RunTabView
+            onClickHome={navigateToHome}
+            onClickBack={navigateToTiming}
+            onClickNext={navigateToHome}
```

Comment:
The design for this tab was to have `Back to dashboard` button.
https://www.figma.com/file/1Lv4r46oF1ioHZ6a9JRaRR/World-Admin?node-id=0%3A1

The is also a `Save` button but I have no idea what it's supposed to save, because there is nothing on that page that needs saving.

Reply:
It's a bit of a mess, currently the save and cancel are inside with the form displayed when editing portals. That form is OOS to this PR and should be dealt with in a subsequent one.